relayer/pkg/chainlink/keys/ocr2key.go

```diff
@@ -16,25 +16,25 @@ import (
 	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2/types"
 )
 
-var _ ocrtypes.OnchainKeyring = &OCR2Keyring{}
+var _ ocrtypes.OnchainKeyring = &OCR2Key{}
 
-type OCR2Keyring struct {
+type OCR2Key struct {
```

Comment:
Do we need this public?

Reply:
yeah we do. it's used as part of defining generics - so we need the type to properly set up the generic struct